redis/redis.go

```diff
@@ -52,85 +115,46 @@ type Commands interface {
 	SRandMember(key string, count int) ([]string, error)
 	SDiff(key string, keys ...string) ([]string, error)
 	SIsMember(key string, member string) (bool, error)
-
-	// Sorted Sets - http://redis.io/commands#sorted_set
-
-	ZAdd(key string, score float64, value string) (int, error)
-	ZCard(key string) (int, error)
-	ZRangeByScore(key, start, stop string, options ...interface{}) ([]string, error)
-	ZRangeByScoreWithLimit(key, start, stop string, offset, count int) ([]string, error)
-	ZRem(key string, members ...string) (removed int, err error)
-	ZIncBy(key string, score float64, value string) (int, error)
-
-	// HyperLogLog
-
-	PFAdd(key string, values ...string) (int, error)
-	PFCount(key string) (int, error)
-	PFMerge(mergedKey string, keysToMerge ...string) (bool, error)
 }
 
-type NoResultCommands interface {
```

Comment:
What happened to this interface? Is it now called `BatchCommands`? If so, should there be an alias to `NoResultCommands` for backward compatibility?